### ISSUE TITLE
Contributor suggestions

### ISSUE THREAD

**Issue by MV10** [opentk/opentk#1654]
A few things I ran into while preparing to work on a fix as a newcomer to working on OpenTK itself:

* The ["Contributing" wiki page](https://github.com/opentk/opentk/wiki/Contributing) page is badly outdated and incomplete. The [`contributing.md`](https://github.com/opentk/opentk/blob/master/CONTRIBUTING.md) document in the solution's project folder is much more useful.

* The "Setting Up" section of the [`contributing.md`](https://github.com/opentk/opentk/blob/master/CONTRIBUTING.md) document should clarify that Visual Studio can't build or run anything (such as the basic `LocalTest` project) before a command-line build is run at least once -- possibly with some warnings about the next point.

* The first step of the Windows [build.cmd](https://github.com/opentk/opentk/blob/master/build.cmd) script is to run a CI-oriented powershell installer for the .NET 6.0 SDK. Unfortunately that installer will _replace_ any globally-installed matching SDK (ie. one that is installed under `C:\ProgramFiles\dotnet` either by Visual Studio itself or a downloadable installer) with a so-called "local" install in the user's `AppFiles` path. The local install isn't seen by Visual Studio, so after that change, VS builds still fail with `NETSDK1141` errors (`Unable to resolve the .NET SDK version as specified in the global.json`). The only fix is to manually delete the "local" install and run the downloadable SDK installer again, which isn't very obvious. If `build.cmd` is primarily for CI scenarios, either a switch should be added, or a different "setting up" build script should be provided, since a command-line build is required before the solution will work in VS. If it isn't primarily for CI builds, it should warn and prompt before continuing with the "local" install. (Not sure about this, but it appears the "local" install may be necessary for FAKE to work; I'm not familiar with FAKE but I saw some complaints about that requirement from several years back.)

I'm happy to PR something for the last two, depending on the intent of `build.cmd` (primarily for CI, or not?)

**Comment by NogginBops** [comment]
The contributor documentation is very out of date. The goal has been to make it as seamless as possible.
One part of that was removing the need to run the `build.cmd` script before being able to build in VS (it was done in #1520), if you experienced any issues with this I would appreciate if you could open up a separate issue with the details there.

Any updates to the documentation is welcome, so if you feel like editing these documents (even with just a warning message that the documents are old) that would be appreciated. 🙂

**Comment by MV10** [comment]
I added a link to the contributing doc to the wiki, so I'll close this.

Once that Win32 PR is done, I'll wipe everything and start from scratch and open a new issue about first-time builds / errors.
